servers/physics_2d/broad_phase_2d_hash_grid.h

```diff
@@ -115,8 +118,15 @@ class BroadPhase2DHashGrid : public BroadPhase2DSW {
 	UnpairCallback unpair_callback;
 	void *unpair_userdata;
 
-	void _enter_grid(Element *p_elem, const Rect2 &p_rect, bool p_static);
-	void _exit_grid(Element *p_elem, const Rect2 &p_rect, bool p_static);
+	_FORCE_INLINE_ bool _test_collision_mask(uint32_t mask1, uint32_t layer1, uint32_t mask2, uint32_t layer2) {
```

Comment:
For these two inline methods:
They could probably be merged into just one, since the first one is only called in the second one.
Arguments should use `p_` prefix (coding style).
Could be static since it's not using any member (mostly to make things clear).

Reply:
Done, thanks